src/views/Verify/index.view.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react"
+import "./index.scss"
+import { useAuth0 } from "@auth0/auth0-react"
+import { resendVerificationEmail } from "../../utils/Api"
+
+const Verify: React.FC = () => {
+  const { user } = useAuth0()
+  return (
+    <>
+      <div className='verify-container'>
+        You are not a verified user
+        <button
+          type='button'
+          onClick={() => resendVerificationEmail(user?.sub)}
```

Comment:
Is this supposed to POST to the current url? I'm getting a 404 error and it doesn't resend the verification email.

Reply:
So this is something we need to do in the backend, I just created a mock function for now to eventually be routed to our backend function. This is something that will be done in our next sprint.